schema.json

```diff
@@ -610,12 +719,17 @@
             "type": "string"
           }
         },
+        "required": [
+          "name",
+          "proficiency"
```

Comment:
Interesting that you require proficiency for skills, but not for language. Why?

Reply:
My bad. I will mark the profiency in language a requirement too. A language or skill without a porficency would not provide enough information.